docs/faq/client.md

```diff
@@ -0,0 +1,14 @@
+# FAQ: LeoFS Clients
+
+<!-- toc -->
+
+## Name Resolution
+
+* There are two ways to access LeoGateway from LeoFS Clients.
+    * Path Style Access
+    * Sub-Domain Style Access
+* If a LeoFS Client you use adopts the Sub-Domain Style Access then the client need to resolve **bucket.endpoint** domain into the ip address of LeoGateway.
```

Comment:
As well as this line:


If a LeoFS' client you use adopts the sub-domain style access, then the client need to resolve **bucket.endpoint** into the IP address of LeoGateway.